synthtool/gcp/samples.py

```diff
@@ -0,0 +1,83 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import glob
+import re
+import os
+import yaml
+from typing import List, Dict
+
+
+def _read_sample_metadata_comment(sample_file: str) -> Dict:
```

Comment:
The PR describes the new code as duplicating existing logic.  Where is the existing logic?

Reply:
It's currently in the [`common.CommonTemplates` class](https://github.com/googleapis/synthtool/blob/2e9bcac85a01d7d4fcc9fd0b06d80f75207e01f3/synthtool/gcp/common.py#L144-L168)

Reply:
Note that the original is removed in the part 2 draft linked in the description.